node/src/test/test-DirectTransport.ts

```diff
@@ -138,7 +144,7 @@ test('dataProducer.send() succeeds', async () => {
 			);
 		});
 
-		sendNextMessage();
+		await sendNextMessage();
```

Comment:
I have some hope that this will fix this CI random failure: https://github.com/versatica/mediasoup/actions/runs/7448459449/job/20262892577?pr=1305